src/pages/Game/GamePage.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useRef, useEffect } from 'react';
+import { Container } from '@mui/material';
+import { Game } from '../../components/game/Game';
+import './GamePage.scss'
+
+const GamePage: React.FC<{}> = () => {
```

Comment:
Кажется, не лучший вариант типизации.

Reply:
❗️
Не нужно передавать дженерик, если смысла в нём нет
```suggestion
const GamePage: React.FC = () => {
```

Reply:
Убрал!